src/codegen.cpp

```diff
@@ -7159,10 +7159,11 @@ extern "C" void jl_init_codegen(void)
     initializeAnalysis(Registry);
 #endif
 
-    InitializeNativeTarget();
-    InitializeNativeTargetAsmPrinter();
-    InitializeNativeTargetAsmParser();
-    InitializeNativeTargetDisassembler();
+    InitializeAllTargets();
```

Comment:
maybe make this conditional on building with polly for now, since that's a compile-time setting, if that's the only use case